src/main/resources/swagger.api/schemas/d2ir/borrowerRenewDTO.json

```diff
@@ -0,0 +1,14 @@
+{
+  "description": "Borrower Renew Message for Item Hold",
+  "type": "object",
+  "properties": {
+    "dueDateTime": {
+      "description": "Due Date time",
+      "type": "integer"
+    }
+  },
+  "additionalProperties": false,
+  "required": [
+    "dueDateTime"
+  ]
+}
```

Comment:
I suppose fields from BaseCircRequestDTO (transactionTime, patronId, patronAgencyCode, itemAgencyCode, itemId) should also be included here, but was not mentioned in the task description. Could you please clarify this with PO?

Reply:
yep, documentation says that fields from BaseCircRequestDTO are mandatory. Added